litestar/config/response_cache.py

```diff
@@ -49,6 +71,9 @@ class ResponseCacheConfig:
     """:class:`CacheKeyBuilder <.types.CacheKeyBuilder>`. Defaults to :func:`default_cache_key_builder`."""
     store: str = "response_cache"
     """Name of the :class:`Store <.stores.base.Store>` to use."""
+    do_cache_predicate: Callable[[HTTPScope, int], bool] = field(default=default_do_cache_predicate)
```

Comment:
Maybe `should_cache_response` would be a better name for this? I know technically the `predicate` is more precise, but this feels more intuitive to me

Reply:
Yeh, I hate the name `do_cache_predicate` too.. still used it though :smile:

Reply:
WDYT about `cache_filter`?

Reply:
Well, I went with `cache_filter` for now, I think it is succinct and fairly obvious what it is.. happy to change again if you don't like it.

Reply:
Hmm. That kinda sounds like it would filter in the other direction, i.e. checking if an already cached response should be used

Reply:
`cache_response_filter`?